getting-started.md

```diff
@@ -0,0 +1,142 @@
+```
+
+It will then prompt you to **input a passphrase**, and then **confirm that passphrase**. You'll use this username and passphrase to login to Vega Console. (Instructions on connecting to Console are below.)
+
+The genkey command in that instruction will generate public and private keys for the wallet, at the same time as creating a user name.
+
+You’ll see an output with a "mnemonic" and a public and private key. DO NOT SHARE YOUR MNEMONIC OR YOUR PRIVATE KEY.
+
+**The mnemonic acts as a backup key, from which the wallet can restore all your keys.** Keep it safe and secret. If lost, you won't be able to retrieve your keys. Anyone who has this mnemonic will be able to use your keys.
+
+You don’t need to save your private key, as you’ll be able to retrieve it from your Wallet in the future.
+
+> Tip: You can give each new key a nickname/alias. 
+> When creating a key, run 
+> 
+> MacOS & Linux: `./go-wallet genkey -name="YOUR_CUSTOM_USERNAME" --metas="name:CHOOSE_CUSTOM_ALIAS_FOR_KEY"`. 
+> 
+> Windows: `go-wallet genkey -name="YOUR_CUSTOM_USERNAME" --metas="name:CHOOSE_CUSTOM_ALIAS_FOR_KEY"`
+
+> Tip: To give an existing key a nickname/alias, run 
+> 
+> MacOS & Linux: `./go-wallet meta --metas="name:CHOOSE_CUSTOM_ALIAS_FOR_KEY" --name="YOUR_CUSTOM_USERNAME" --pubkey="REPLACE_THIS_WITH_YOUR_PUBLIC_KEY"`. 
+> 
+> Windows: `go-wallet meta --metas="name:CHOOSE_CUSTOM_ALIAS_FOR_KEY" --name="YOUR_CUSTOM_USERNAME" --pubkey="REPLACE_THIS_WITH_YOUR_PUBLIC_KEY"`
+
+> Tip: You can also use the meta command to tag a key with other data you might want, using a property name and a value. This will be useful for developing with Vega Wallet in the future.
+
+## Run the Wallet service
+Now, **connect your Wallet to the Testnet nodes**. The `init` command (below) will initialise the configuration. A configuration file will be stored in your home folder, in a folder called `.vega`.
+
+**MacOS & Linux**
+
+```console
+./go-wallet service init
+```
+**Windows**
+
+```console
+go-wallet service init
+```
+
+> Tip: If you want to specify a root-path, it will not go into the default path, but a folder you choose to create. If you want to create a new config for a new wallet, or test or isolate it, you should specify the root path.
+
+Next: To trade, run the wallet and **start the Vega Console** with the command below. (You'll need collateral to trade, and you can deposit it through Vega Console, once you're connected.)
+
+**MacOS & Linux**
+
+```console
+./go-wallet service run -p
+```
+**Windows**
+
+```console
+go-wallet service run -p
+```
+
+> Tip: If you're running an ad/tracker blocker, and you're getting errors, it may be blocking the node from connecting. Try allowlisting lb.testnet.vega.xyz and refreshing.
+
+> Tip: To terminate the process, such as if you want to run other commands in Wallet, use ctrl+c.
+
+> Tip: See a full list of available commands in [the cmd readme](/cmd/README.md).
+
+### Create and deposit testnet tokens
```

Comment:
I added this part to the CLI documentation, as well. Should we remove it from there, and keep yours ? I think that would avoid us to maintain duplicated explanation, and I think that's fairly probable the users will have a look to getting started first.

Reply:
I wasn't sure what to do about that, actually. I think you're right, and if people start telling us they can't find instructions for depositing then we can reinstate it. I'll remove from the other location.